src/extractors/custom/deadline.com/index.js

```diff
@@ -0,0 +1,34 @@
+export const DeadlineComExtractor = {
+  domain: 'deadline.com',
+
+  title: {
+    selectors: ['h1'],
+  },
+
+  author: {
+    selectors: ['section.author h3'],
+  },
+
+  date_published: {
+    selectors: [['meta[name="article:published_time"]', 'value']],
+  },
+
+  dek: null,
+
+  lead_image_url: {
+    selectors: [['meta[name="og:image"]', 'value']],
+  },
+
+  content: {
+    selectors: ['div.a-article-grid__main.pmc-a-grid div.pmc-a-grid-item'],
```

Comment:
The structure of the website has slightly changed since you last created this custom extractor, and this selector no longer matches the content correctly. I have noticed that `div.pmc-a-grid-item` has changed into `article.pmc-a-grid-item`. Could you please update this with a suitable and optimized selector that might be less prone to breaking?

Reply:
Thank you.
I fixed the content selector.